### PR TITLE
backstage: use Origami Build Service v3

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: views/partials/html-head.html [hunk 1]
```diff
@@ -18,22 +18,12 @@
 	.enhanced .o--if-no-js {
 		display: none !important;
 	}
-	html {
-		font-family: BentonSans, sans-serif;
-		overflow-x: hidden;
-	}
```
Comment (chee) [inline views/partials/html-head.html:24]: lol nice

File: views/partials/html-head.html [hunk 2]
```diff
@@ -18,22 +18,12 @@
...
-	body {
-		margin: 0;
-	}
 	.o-layout__header .o-header-services__top .o-header-services__ftlogo {
 		background-image: url("https://www.ft.com/__origami/service/image/v2/images/raw/ftlogo-v1:origami?source=origami-navigation-service");
 	}
-	.o-table--row-headings tbody th {
-		width
-	}
 </style>
```
Comment (chee) [inline views/partials/html-head.html:24]: width

**Review by chee — APPROVED** [review]
width

**Comment by notlee** [comment]
width? width!!

**Pull request merged**